tests/e2e/zookeeper_test.go

```diff
@@ -92,9 +118,9 @@ func requireRemoveZookeeperOperatorCRDs(kubectlOptions *k8s.KubectlOptions) {
 func requireZookeeperClusterReady(kubectlOptions *k8s.KubectlOptions) {
 	It("Verifying Zookeeper cluster health", func() {
 		By("Verifying the Zookeeper cluster resource")
-		waitK8sResourceCondition(kubectlOptions, zookeeperCRDs()[0], "jsonpath={.status.readyReplicas}=1", "240s", "", zookeeperClusterName)
+		waitK8sResourceCondition(kubectlOptions, zookeeperCRDs()[0], "jsonpath={.status.readyReplicas}=1", zookeeperClusterCreateTimeout, "", zookeeperClusterName)
```

Comment:
I think it is better to use here: https://github.com/banzaicloud/koperator/blob/ef6db9416dc459e992587ad259c00da1ac2fa130/tests/e2e/const.go#L37
I dont like this zookeeperCRDs()[0] :)